common/enclave/cc_data.h

```diff
@@ -45,6 +45,9 @@ class cc_data
     ByteArray get_state_encryption_key();
     std::string get_enclave_id();
     bool sign_message(const ByteArray& message, ByteArray& signature) const;
+    bool decrypt_cc_message(const ByteArray& encrypted_message, ByteArray& message) const;
```

Comment:
the `_cc_` seems a bit inconsistent given that we don't do it for sign and encrypt ...

Reply:
I kind of agree.
Thing is that `encrypt_message` uses just symmetric crypto and passes a key, while `decrypt_cc_message` uses (for now just) asymmetric crypto.
So the "inconsistency" aims actually at differentiating this aspect.

Reply:
not that the name does reflect this distinction? (for that i would have expected something like `_sym_` and `_pk_` or alike).  BTW: also realize now that given that you pass the key (rather taking it from object like for decrypt), it's also less obvious why it is even a method of that object?  Arguably, one could also have set the return key as object field, in which case arguably the sym vs pk distinction at this level of abstraction is actually completely irrelevant?  All that said, the current naming certainly shouldn't lead to any confusion, so no real issue by leaving it as-is, just a bit nitpicking ;-)